webpage_of_me/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://fonts.googleapis.com/css?family=Muli:400,800" rel="stylesheet">
+        <title>About Me</title>
+    </head>
+    <body>
+        <nav class="navigation">
+            <ul>
+                <li class="nav_button"><a class="selected" href="index.html">About Me</a></li>
+                <li class="nav_button"><a href="projects.html">Projects</a></li>
+                <li class="nav_button"><a href="resume.html">Resume</a></li>
+            </ul>
+        </nav>
+        <section class="bio">
+            <img class="photo_me" src="images/profile_photo.jpg" alt-text="Headshot of Alyssa">
```

Comment:
👏 accessibility! I like that you included the text descriptor. This might be an old spec though? All you need is `alt = "description"`